src/khoj/processor/conversation/prompts.py

```diff
@@ -120,16 +120,21 @@
 
 image_generation_improve_prompt = PromptTemplate.from_template(
     """
-You are a talented creator. Generate a detailed prompt to generate an image based on the following description. Update the query below to improve the image generation. Add additional context to the query to improve the image generation. Make sure to retain any important information from the query. Use the conversation log to inform your response.
+You are a talented creator. Generate a detailed prompt to generate an image based on the following description. Update the query below to improve the image generation. Add additional context to the query to improve the image generation. Make sure to retain any important information originally from the query. You are provided with the following information to help you generate the prompt:
 
 Today's Date: {current_date}
 User's Location: {location}
 
+User's Notes: {references}
+
+Online References: {online_results}
+
```

Comment:
nit: Should references be on a new line as they're multi-line strings like the conversation log

```suggestion
User's Notes:
{references}

Online References:
{online_results}

```